gluster/cli/utils.py

```diff
@@ -38,96 +53,105 @@ def execute_or_raise(cmd):
 
 
 def gluster_system_execute(cmd):
-    cmd = [GLUSTERCMD, "--mode=script", "system::", "execute"] + cmd
+    cmd.append("system::")
+    cmd.append("execute")
```

Comment:
Wrong modification, This func is called from Geo-rep https://github.com/gluster/glustercli-python/blob/master/gluster/cli/georep.py, For example

    gluster_system_execute(["gsec_create"])

This should be converted as 

    gluster --mode=script system:: execute gsec_create

But with the above change, this will become

    gluster --mode=script gsec_create system:: execute

Can be changed as,

    cmd = ["system::", "execute"] + cmd

or `insert(0, "system::")` and `insert(1, "execute")` as in other parts of the code